controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -314,6 +325,30 @@ public OperationContext createStreamContext(String scopeName, String streamName,
         }), executor);
     }
 
+    /**
+     * Delete a scope recursively with given name.
+     *
+     * @param scopeName Name of scope to be deleted
+     * @return DeleteScopeRecursiveStatus future.
+     */
+    @Override
+    public CompletableFuture<DeleteScopeRecursiveStatus> deleteScopeRecursive(final String scopeName, final OperationContext ctx,
```

Comment:
@shshashwat Why is this method returning `DeleteScopeRecursiveStatus` ? The only place where I see it being called is `DeleteScopeTask::execute` method which does not do anything with the return status.

Reply:
we are using this for a logger now and this part will be removed during enhancement